ipod-shuffle-4g.py

```diff
@@ -470,6 +492,9 @@ def set_master(self, tracks):
         self["listtype"] = 1
         self.listtracks = tracks
 
+    def set_podcast(self):
+        self["listtype"] = 3
```

Comment:
Do we really need a function for that single thing?

Reply:
I figured it fit in with the existing `set_master` method and could be useful if other podcast features are added in the future. It is kinda bulky as is though. I'll remove it